modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties

```diff
@@ -2244,6 +2244,11 @@ background-state-info=Background State Info
 background-state-success=Background State Success
 background-state-warning=Background State Warning
 background-styles=Background Styles
+background-task-manager-configuration-name=Background Task Manager Configuration
+background-task-workers-core-size=Core Processes
+background-task-workers-core-size-help=The number of processes always available for executing background tasks. This setting cannot exceed the max processes setting.
```

Comment:
@vendeltoreki, I think this explanation is not suitable. We use java.util.concurrent.ThreadPoolExecutor to do this. You may refer to corePoolSize and maximumPoolSize of ThreadPoolExecutor to write the explanation.

Reply:
@yuhai, thanks for the suggestions! About this language constant, I copied it mostly from `workers-core-size` (see [Language.properties](https://raw.githubusercontent.com/liferay/liferay-portal/master/modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties) ). It basically sets the the same parameter in AM background processes: [AMConfiguration.java](https://github.com/liferay/liferay-portal/blob/c095d83e59e09f7f1e439efaba10a01a7b2e8785/modules/apps/adaptive-media/adaptive-media-web/src/main/java/com/liferay/adaptive/media/web/internal/configuration/AMConfiguration.java#L48).
Should I change the configuration method names and language keys as well, or only the explanation (value of `background-task-workers-core-size-help`)?

Reply:
@vendeltoreki, Refer to DestinationWorkerConfiguration and ThreadPoolExecutor, I think using "threads" or "worker threads" are better than "processes". Does this look good to you?